.storybook/types/global.js

```diff
@@ -77,14 +64,18 @@ export default {
 	lang: {
 		title: "Language",
 		description: "Language of the content",
-		defaultValue: "en-US",
+		defaultValue: "en_US",
 		type: "string",
 		toolbar: {
 			items: [
-				{ value: "en-US", title: "🇺🇸", right: "English (US)" },
-				{ value: "ja", title: "🇯🇵", right: "日本語" },
-				{ value: "ko", title: "🇰🇷", right: "한국어" },
-				{ value: "zh", title: "🇨🇳", right: "中文" },
+				{ value: "en_US", title: "English", right: "English (US)" },
+				{ value: "he", title: "Hebrew", right: "עִברִית" },
+				{ value: "ja", title: "Japanese", right: "日本語" },
+				{ value: "ko", title: "Korean", right: "한국어" },
+				{ value: "ar", title: "Arabic", right: "عربي" },
+				{ value: "zh", title: "Chinese", right: "中文" },
+				{ value: "fa", title: "Persian", right: "فارسی" },
```

Comment:
I did not see Persian on the list of [supported locales](https://react-spectrum.adobe.com/react-aria/internationalization.html#supported-locales), we certainly still could support it though!